packages/docs/docs/guides/6-hosting-a-node/3-helm.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deploying a node in Kubernetes with Helm
+keywords: [Request node, helm, kubernetes]
+---
+
+Deploying a Request node on Kubernetes is straightforward using our [helm](https://helm.sh/) chart.
+
+You can see our chart in our [git repository](https://github.com/RequestNetwork/request-helm-charts/tree/master/request-node).
+
+## Adding the chart
+
+We host our chart on our own helm repo, since we upgrade it frequently.
+To add our chart you can run:
+
+```bash
+helm repo add request https://request-charts.storage.googleapis.com
+helm repo update
+```
+
+## Installing the chart
+
+To install our chart with the release name `my-release`, you can run:
+
+```bash
+helm install --name my-release request/request-node
```

Comment:
Does this command run the Request Node ?

If not we need an instruction to run the Request Node with Helm

Reply:
this will deploy the request node in kubernetes.